#885: Update README, inform about reviewing (#887)
agda/cubical: README.md

If you want to use Agda 2.6.0.1 instead of the latest release version, you
can check out the tag `v0.1` of this library.

> Learning materials

Reviewer: We should add the HoTTest summer school material here: https://github.com/martinescardo/HoTTEST-Summer-School/tree/main/Agda/Cubical and https://www.youtube.com/channel/UC-9jDbJ-HegCFuWuam1SfvQ

Author: Added

This makes it possible to directly represent higher inductive types.

Reviewing of pull requests
--------------------------
If you switch your draft pull request (PR) to 'ready to merge',
or directly create an open PR,
we should request a review, by one of the reviewers below.

> If that doesn't happen, you can also request a review yourself,

Reviewer: ```suggestion
If that doesn't happen, you can also request a reviewer yourself,
```

> If that doesn't happen, you can also request a review yourself,

Reviewer: ```suggestion
If that doesn't happen, you can also request a reviewer yourself (for reviewer expertise see below),
```

Author: Yes -> done

to make us aware of the open PR.

> * [Anders Mörtberg](https://staff.math.su.se/anders.mortberg/)

Reviewer: ```suggestion
* [Anders Mörtberg](https://staff.math.su.se/anders.mortberg/) (*Most topics*)
```

> * [Evan Cavallo](https://staff.math.su.se/evan.cavallo/)

Reviewer: @ecavallo Any preferred topics?

Reviewer 2: Nothing in particular comes to mind.

> * [Andrea Vezzosi](http://saizan.github.io/) (*inactive*)

Reviewer: Maybe move Andrea to the end of the list if he's inactive?

Author: Yes, good point.
Also, I didn't know that @mzeuner and  @aljungstrom do reviews - they should be on the list then.

Reviewer: I think they both did some after I requested them to do it :-)

Author: I added them. Any hints for their topics?

Reviewer: algebra for Max and synthetic homotopy/cohomology theory for Axel

Author: Added

> * [Felix Cherubini](https://felix-cherubini.de) (*Limited roughly to algebra related topics*)

Reviewer: ```suggestion
* [Felix Cherubini](https://felix-cherubini.de) (*Mainly algebra related topics*)
```

Reviewer: How about modalities? That's definitely more your cup of tea than mine...

This makes it possible to directly represent higher inductive types.

Reviewing of pull requests
--------------------------
If you switch your draft pull request (PR) to 'ready to merge',
or directly create an open PR,
we should request a review, by one of the reviewers below.
If that doesn't happen, you can also request a reviewer yourself (for reviewer expertise see below),

> to make us aware of the open PR.

Reviewer: ```suggestion
to make us aware of the open PR. Feel free to use Discord to get in touch with a reviewer in case reviewing is taking a very long time.
```

Author: Ah - that's good!
